tests/test_blueprints.py

```diff
@@ -893,3 +893,64 @@ def third(request):
 
     assert app.test_client.get("/f1")[1].status == 200
     assert app.test_client.get("/f1/")[1].status == 200
+
+
+def test_blueprint_group_versioning_and_strict_slash():
```

Comment:
I think versioning and strict_slashes should really be tested separately. Will make refactoring and maintenance easier.

Reply:
👍